### PR TITLE
Fix deploy to Node.js hostings

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**

### CODE REVIEW

**Comment by FluorescentHallucinogen** [comment]
Not sure about minimum required Node.js version in `packages.json`. Feel free to edit it. ;)

File: src/demos/node-server/index.js [hunk 1]
```diff
@@ -208,6 +208,8 @@ app.post('/api/trigger-push-msg/', function (req, res) {
   /**** END trig-push-return-response ****/
 });
 
-const server = app.listen(9012, function () {
-  console.log('Running on http://localhost:' + server.address().port);
+const port = process.env.PORT || 9012;
+
+const server = app.listen(port, function () {
+  console.log('Running on ' + server.address().port + ' port');
```
Comment (gauntface) [inline src/demos/node-server/index.js]: Could you add in the host-name so new users have a URL to copy into the browser.

**Comment by gauntface** [comment]
Thanks for the PR, one minor change request just to make it accessible to people who might be new to node and / or the project.

**Comment by gauntface** [comment]
Oh and if you'd like to add yourself to the contributor list that would be great too :)

https://github.com/gauntface/web-push-book/blob/master/src/contributors.md

File: src/demos/node-server/index.js (hunk 1 above)
Comment (FluorescentHallucinogen) [inline src/demos/node-server/index.js]: Done. ;)

**Pull request merged**